docs/version0.44.md

```diff
@@ -32,6 +32,7 @@ The following new features and notable changes since version 0.43.0 are included
 - [New `-XX:[+|-]ShowUnmountedThreadStacks` option added](#new-xx-showunmountedthreadstacks-option-added)
 - [VMID query in the `jcmd` tool enhanced](#vmid-query-in-the-jcmd-tool-enhanced)
 - [DDR field names in `J9BuildFlags` changed](#ddr-field-names-in-j9buildflags-changed)
+- [Deletion of the files within the `/tmp/` folder by `systemd-tmpfiles` controlled](#deletion-of-the-files-within-the-tmp-folder-by-systemd-tmpfiles-controlled)
```

Comment:
If I say " Deletion of the files within the `/tmp/` folder by `systemd-tmpfiles` **is** controlled" , is it fine? Or do you have an issue with the word "controlled"? I have used this word because we are controlling the deletion of files within the /tmp folder, isn't it?

Reply:
"is controlled" is the proper grammar but I also have an issue with the meaning, as it seems cryptic. I think we want to say something along the lines of "Deletion of files within the `/tmp/` folder by `systemd-tmpfiles` is avoided and configurable".

Reply:
How about "New system property added to prevent the deletion of the Attach API  control files within the `/tmp/` folder"?

Reply:
Sounds good.